cinema/serializers.py

```diff
@@ -132,6 +149,31 @@
                 Ticket.objects.create(order=order, **ticket_data)
             return order
 
+    def update(self, instance, validated_data):
+        tickets_data = validated_data.pop("tickets")
+        instance = super().update(instance, validated_data)
+
+        keep_tickets = []
+        for ticket_data in tickets_data:
+            if "id" in ticket_data.keys():
+                if Ticket.objects.filter(id=ticket_data["id"]).exists():
+                    ticket = Ticket.objects.get(id=ticket_data["id"])
+                    ticket.row = ticket_data.get("row", ticket.row)
+                    ticket.seat = ticket_data.get("seat", ticket.seat)
+                    ticket.save()
+                    keep_tickets.append(ticket.id)
+                else:
+                    continue
```

Comment:
The logic for updating existing tickets could be improved. Instead of checking if the ticket exists twice (once with `filter` and once with `get`), you can use `get` with `try-except` to handle the `DoesNotExist` exception, which is more efficient.